### DISCUSSION TITLE
pathom-viz showing #object tags

### DISCUSSION THREAD

**Discussion by wardle** [wilkerlucio/pathom3#39] [Q&A]
Hello. I'm getting an odd response in pathom-viz that I don't get at the REPL.
For a (seemingly) random set of properties, they get an `#object` tag:

<img width="1200" alt="image" src="https://user-images.githubusercontent.com/286951/115268478-38e68400-a132-11eb-8577-75e078cbf1ec.png">

But at the REPL, it works perfectly:
```
(p.eql/process registry {:id 80146002} [:info.snomed.Concept/id
                                          :info.snomed.Concept/active
                                          '(:info.snomed.Concept/preferred-description {:accept-language "en-GB"})
                                          :info.snomed.Concept/refsets
                                          {:info.snomed.Concept/descriptions
                                           [:info.snomed.Description/active :info.snomed.Description/term]}])
```

```
#:info.snomed.Concept{:id 80146002,
                      :active true,
                      :preferred-description #:info.snomed.Description{:caseSignificanceId 900000000000448009,
                                                                       :moduleId 900000000000207008,
                                                                       :typeId 900000000000013009,
                                                                       :effectiveTime #object[java.time.LocalDate 0x1a5944be "2017-07-31"],
                                                                       :active true,
                                                                       :id 132973012,
                                                                       :conceptId 80146002,
                                                                       :term "Appendicectomy",
                                                                       :languageCode "en"}
```

Am I doing something wrong or is this an issue within pathom-viz? 

Mark

PS. This is amazing work. I am very excited to federate across multiple services with pathom3 as the core abstraction between client and server. I used Apple's WebObjects for a long time and it is a bit like a super-duper key value coding, which of course came from NeXTSTEP - used to decouple the UI from the backend - and it's not dissimilar to OGNL - except much much better!

**Comment by wilkerlucio** [comment]
Hello Mark, glad to hear you are having a good time with Pathom :)

About the tools, for looking at the numbers that turn wrapped into #object, my guess is they are bigints, which use some special transit types to encode. So what you see is the number wrapped by `com.cognitect.transit.BigInteger`, which is an object, and that causes the print to look like that.

**Reply by wardle** [reply]
Makes sense! These are longs so I will happily ignore the printing! Thanks again!
